pkg/controller/compliancescan/rawresults.go

```diff
@@ -0,0 +1,80 @@
+package compliancescan
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	compv1alpha1 "github.com/openshift/compliance-operator/pkg/apis/compliance/v1alpha1"
+	"github.com/openshift/compliance-operator/pkg/controller/common"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// handles the necessary things to store and expose the raw results from the scan. This implies
+// that the PVC gets created, and, if necessary, the scan instance will get updated too.
+// Returns whether the reconcile loop should continue or not, and an error if encountered.
+func (r *ReconcileComplianceScan) handleRawResultsForScan(instance *compv1alpha1.ComplianceScan, logger logr.Logger) (bool, error) {
+	// Create PVC
+	pvc := getPVCForScan(instance)
+	logger.Info("Creating PVC for scan", "PersistentVolumeClaim.Name", pvc.Name, "PersistentVolumeClaim.Namespace", pvc.Namespace)
+	if err := r.client.Create(context.TODO(), pvc); err != nil && !errors.IsAlreadyExists(err) {
+		return false, err
+	}
```

Comment:
btw do you think creating a PVC is something that should issue an event?

Reply:
it already does. But not on the compliancescan. I could do that on a subsequent iteration.